src/pages/CardRegistrationConfirmation/CardRegistrationConfirmation.tsx

```diff
@@ -0,0 +1,55 @@
+import CardPreview from '../../components/CardPreview/CardPreview';
+import styles from './CardRegistrationConfirmation.module.css';
+import CardNicknameInput from '../../components/CardNickname/CardNicknameInput';
+import Button from '../../components/Button/Button';
+import { useNavigate } from 'react-router-dom';
+import { CardInfo } from '../../types';
+import { useCardStore } from '../../hook/useCardState';
+
+type CardRegistrationConfirmationProps = {
+  registerNewCard: (cardInfo: CardInfo) => void;
+};
+
+const CardRegistrationConfirmation = ({ registerNewCard }: CardRegistrationConfirmationProps) => {
+  const navigate = useNavigate();
+  const { get } = useCardStore();
+  const cardNumber = get().cardNumber;
+  const cardOwnerName = get().cardOwnerName;
+  const expirationDate = get().expirationDate;
+  const selectedCard = get().selectedCard;
+  const cardNickName = get().cardNickName;
+
+  const handleCardInfo = (e: React.MouseEvent<HTMLButtonElement>) => {
+    const cardInfo: CardInfo = {
+      cardNumber: cardNumber,
+      expirationDate: expirationDate,
+      cardOwnerName: cardOwnerName,
+      selectedCard: selectedCard,
+      cardNickName: cardNickName,
+    };
+
+    registerNewCard(cardInfo);
+
+    navigate('/');
+  };
```

Comment:
이 형태도...

```suggestion
  const handleCardInfo = useCallback((e: React.MouseEvent<HTMLButtonElement>) => {
    const cardInfo: CardInfo = { ...get() };

    registerNewCard(cardInfo);

    navigate('/');
  }, []);
```

이렇게 되면 어떨까요?

또 CardPreview 에서 `useCardStore`를 호출하고 있으므로 필요한 props 일까 싶네요!

Reply:
오홋! 스프레드 연산자와 구조분해 할당을 통해 코드를 간결하게 만들 수 있겠네용! 

그리고 키와 값이 같으니 굳이 일일이 나타낼 필요가 없군여 ㅎㅎ

제안해주신대로 수정 해보았습니당 ^^